Source/ResourceLib/Accelerator.cs

```diff
@@ -45,12 +45,45 @@ public string Key
             get
             {
                 string key = Enum.GetName(typeof(User32.VirtualKeys), _accel.key);
-                return string.IsNullOrEmpty(key) ? ((char) _accel.key).ToString() : key;
+                return string.IsNullOrEmpty(key) ? ((char)_accel.key).ToString() : key;
+            }
+            set
+            {
+                if(Enum.IsDefined(typeof(User32.VirtualKeys), value))   // if given string is a Virtual Key e.g. "VK_NUMPAD1"
+                {
+                    uint i = (uint)Enum.Parse(typeof(User32.VirtualKeys), value);
+                    _accel.key = (UInt16)i;
+                }
+                else    // otherwise, use ASCII-code of given Key e.g. key "Z" == 90
+                {
+                    char c = value.ToCharArray()[0];
+                    UInt16 i = (UInt16)c;
+                    _accel.key = i;
+                }
+            }
+        }
+
+        /// <summary>
+        /// AcceleratorVirtualKey representation of the Accelerator bit-flags.

```

Comment:
Hm, maybe I'm not seeing it or I wrote it wrong but for me the comment seems correct.
By the way I'm not sure if this isn't too detailed for a summary? 
I have no experience in that stuff...